tests/harness/cfg/requirements.yaml

```diff
@@ -0,0 +1,203 @@
+#     be set for the stack component to be accepted. The values may reference
+#     secrets that are defined in the test configuration or as environment
+#     variables using the `{{SECRET_NAME_OR_ENV_VAR}}` syntax.
+# - capabilities: A list of custom capabilities that the entry either provides
+#   (when used or referenced in an environment) or requires (when used or
+#   referenced in a test). Test capability requirements are compared against the
+#   capabilities provided by the active environment and deployment to determine
+#   if they can be run. When used or referenced in an environment, a `true`
+#   value indicates that the environment supports the capability, while a
+#   `false` value indicates that it doesn't. When used or referenced in a test
+#   configuration , a `true` value indicates that the test requires the
+#   capability to run, while a `false` value indicates that the test cannot run
+#   if the capability is present.
+#
+# Note that the integrations, packages and system tools requirements are not
+# installed automatically. The user is expected to install them in the host OS
+# and/or current virtual environment. The test framework only checks for their
+# presence and fails the environment setup (when used or referenced in
+# environments) or skips the test (when used or referenced in tests) if they are
+# not installed.
+#
+# Example:
+#
+#  ```
+#  requirements:
+#    - name: example
+#      integrations:
+#        - pytorch
+#        - xgboost
+#      packages:
+#        - numpy==1.19.5
+#        - pandas
+#      system_tools:
+#        - docker
+#        - kubectl
+#      system_os:
+#        - linux
+#        - macos
+#      stacks:
+#        - type: experiment_tracker
+#          flavor: mlflow
+#          configuration:
+#            nested: true
+#        - type: model_deployer
+#          flavor: seldon
+#      capabilities:
+#        mlflow: true
+#        seldon: true
+#      disabled: false
+#  ```
+#
+requirements:
+
+    # Data validators do not require external resources, infrastructure
+    # or services to be set up and can be provisioned on-demand on any
+    # environment and combined with any other stack component (local or remote)
+    # and with any server deployment (local or remote).
+  - name: data-validators
+    stacks:
+      - name: deepchecks
+        type: data_validator
+        flavor: deepchecks
+      - name: evidently
+        type: data_validator
+        flavor: evidently
+      - name: great_expectations
+        type: data_validator
+        flavor: great_expectations
+      - name: whylogs
+        type: data_validator
+        flavor: whylogs
+    capabilities:
+      deepchecks: true
+      evidently: true
+      great_expectations: true
+      whylogs: true
+
+    # Local kubeflow orchestrator components
+  - name: kubeflow-local
+    system_tools:
+      - docker
+      - kubectl
+      - k3d
+    stacks:
```

Comment:
I envision this section may also contain some stacks in the future. This whole story started with the intention of making this a ZenML project descriptor. It may still end up that way, and we may have stacks included in here.

I hope you bought all that, because frankly renaming this now is a whole lot of unnecessary work that I'd rather not have to do :)